ledger-core/v2/virtual/integration/utils/server.go

```diff
@@ -124,7 +125,7 @@ func NewServer(t *testing.T) *Server {
 	virtualDispatcher.Runner = runnerService
 	virtualDispatcher.MessageSender = messageSender
 
-	if err := virtualDispatcher.Init(ctx); err != nil {
+	if err := virtualDispatcher.Init(ctx, &example.MachineLogger{}); err != nil {
```

Comment:
this is wrong, please  copy that structure to integration. We should not use example package